sources/packages/web/src/components/aest/student/PendingAppealsTable.vue

```
@@ -0,0 +1,178 @@
+      type: String as () => "legacy-change-request" | "appeal",
+      required: true,
+    },
+  },
+  setup(props) {
+    const router = useRouter();
+    const { dateOnlyLongString, emptyStringFiller } = useFormatters();
+    const isLoading = ref(false);
+    const searchCriteria = ref("");
+    const applicationAppeals = ref(
+      {} as PaginatedResults<StudentAppealPendingSummaryAPIOutDTO>,
+    );
+
+    const pageTitle = computed(() => {
+      return props.appealsType === "legacy-change-request"
+        ? "Pending change requests"
+        : "Pending appeals";
+    });
+
+    const pageDescription = computed(() => {
+      return props.appealsType === "legacy-change-request"
+        ? "Change requests that require ministry review."
+        : "Appeals that require ministry review.";
+    });
+
+    const tableHeaders = computed(() => PendingChangeRequestsTableHeaders);
+    const gotToAssessmentsSummary = (
+      applicationId: number,
+      studentId: number,
+    ) => {
+      router.push({
+        name: AESTRoutesConst.ASSESSMENTS_SUMMARY,
+        params: {
+          applicationId: applicationId,
+          studentId: studentId,
+        },
+      });
+    };
+
+    const loadAppeals = async (
+      page = DEFAULT_DATATABLE_PAGE_NUMBER,
+      pageLimit = DEFAULT_PAGE_LIMIT,
+      inputSortField?: string,
+      inputSortOrder?: DataTableSortOrder,
+    ) => {
+      try {
+        isLoading.value = true;
+        applicationAppeals.value =
+          await StudentAppealService.shared.getPendingAppeals({
+            page,
+            pageLimit,
+            sortField: inputSortField,
+            sortOrder: inputSortOrder,
+            searchCriteria: {
+              appealType: props.appealsType,
+              searchCriteria: searchCriteria.value,
+            },
+          });
+      } catch (error: unknown) {
+        console.error("Error loading appeals:", error);
+        applicationAppeals.value = { results: [], count: 0 };
```

Isn't this going to fail silently? Don't we want to provide the end user with some kind of an indication that the loading failed instead of simply showing the `toggled` equals to `true` page? Just check with business or other devs please.

---

Yes, it is following same pattern, to display empty if API error.

---

Was the mentioned pattern below copied from some other page?
```ts
console.error("Error loading appeals:", error);
applicationAppeals.value = { results: [], count: 0 };
```

---

Update: Snackbar implemented and console error removed.